ee-feature-pack/common/src/main/resources/modules/system/layers/base/io/jaegertracing/main/module.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="io.jaegertracing">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${io.jaegertracing:jaeger-core}"/>
+<!--        <artifact name="${io.jaegertracing:jaeger-client}"/>-->
+        <artifact name="${io.opentelemetry:opentelemetry-exporter-jaeger}"/>
+        <artifact name="${io.grpc:grpc-okhttp}"/>
+        <artifact name="${io.grpc:grpc-core}"/>
+        <artifact name="${com.google.code.gson:gson}"/>
+        <artifact name="${com.google.android:annotations}"/>
+        <artifact name="${com.squareup.okhttp:okhttp}"/>
+        <artifact name="${com.squareup.okio:okio}"/>
+        <artifact name="${com.google.guava:guava}"/>
+        <artifact name="${com.google.guava:failureaccess}"/>
+        <artifact name="${org.checkerframework:checker-compat-qual}"/>
+        <artifact name="${com.google.j2objc:j2objc-annotations}"/>
+        <artifact name="${com.google.errorprone:error_prone_annotations}"/>
```

Comment:
com.google.errorprone is is a static analysis tool for Java that catches common programming mistakes at compile-time. Compare with findbugs:jsr305 references.

Reply:
See above. :)